webpack.config.ts

```diff
@@ -1,27 +1,29 @@
-/* @flow */
 const webpack = require('webpack')
 const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
 const path = require('path')
-const { version, name, author, license } = require('./package.json')
+const { version, moduleName, author, license } = require('./package.json')
```

Comment:
`moduleName` is this defined ?
Im not fount it.

Reply:
`/*! undefined 0.9.2 | Copyright (c) 2017 UNCOVER TRUTH Inc. | License GPL-3.0 */`